README.md

```diff
@@ -470,11 +470,27 @@ We can combine the two precedent filters:
 
 The argument `-filter withtest` indicates that only commits with at least one modification on test cases are considered.
 
+##Analysis
+In order to perform an analysis of possible repair tools that may have generated commits use the python script at https://github.com/kth-tcs/defects4j-repair-reloaded/tree/comrepair-coming/.
 
+use it as:
+
+``` 
+python analyse_repairability_output.py <path to the json>
```

Comment:
add the line to explain how to produce this JSON?

Reply:
Added. Thanks.